clang/lib/Serialization/ASTReader.cpp

```diff
@@ -9615,7 +9615,7 @@ ModuleFile *ASTReader::getLocalModuleFile(ModuleFile &M, unsigned ID) const {
     // It's a prefix (preamble, PCH, ...). Look it up by index.
     unsigned IndexFromEnd = ID >> 1;
```

Comment:
```suggestion
    int IndexFromEnd = ID >> 1;
```

maybe this is slightly better